packages/postgres-database/src/simcore_postgres_database/migration/versions/763666c698fb_resource_tracker_refactoring.py

```diff
@@ -0,0 +1,140 @@
+    sa.Column('transaction_status', sa.Enum('PENDING', 'BILLED', 'NOT_BILLED', 'REQUIRES_MANUAL_REVIEW', name='credittransactionstatus'), nullable=True),
+    sa.Column('transaction_classification', sa.Enum('ADD_WALLET_TOP_UP', 'DEDUCT_SERVICE_RUN', name='credittransactionclassification'), nullable=True),
+    sa.Column('service_run_id', sa.String(), nullable=True),
+    sa.Column('payment_transaction_id', sa.String(), nullable=True),
+    sa.Column('created', sa.DateTime(timezone=True), server_default=sa.text('now()'), nullable=False),
+    sa.Column('last_heartbeat_at', sa.DateTime(timezone=True), nullable=False),
+    sa.Column('modified', sa.DateTime(timezone=True), server_default=sa.text('now()'), nullable=False),
+    sa.PrimaryKeyConstraint('transaction_id')
+    )
+    op.create_index(op.f('ix_resource_tracker_credit_transactions_product_name'), 'resource_tracker_credit_transactions', ['product_name'], unique=False)
+    op.create_index(op.f('ix_resource_tracker_credit_transactions_service_run_id'), 'resource_tracker_credit_transactions', ['service_run_id'], unique=False)
+    op.create_index(op.f('ix_resource_tracker_credit_transactions_transaction_classification'), 'resource_tracker_credit_transactions', ['transaction_classification'], unique=False)
+    op.create_index(op.f('ix_resource_tracker_credit_transactions_transaction_status'), 'resource_tracker_credit_transactions', ['transaction_status'], unique=False)
+    op.create_index(op.f('ix_resource_tracker_credit_transactions_wallet_id'), 'resource_tracker_credit_transactions', ['wallet_id'], unique=False)
+
+    # Resource tracker wallets credit transations (OLD)
+    op.drop_index('ix_resource_tracker_wallets_credit_transactions_product_name', table_name='resource_tracker_wallets_credit_transactions')
+    op.drop_index('ix_resource_tracker_wallets_credit_transactions_service_run_id', table_name='resource_tracker_wallets_credit_transactions')
+    op.drop_index('ix_resource_tracker_wallets_credit_transactions_transac_110f', table_name='resource_tracker_wallets_credit_transactions')
+    op.drop_index('ix_resource_tracker_wallets_credit_transactions_transac_e117', table_name='resource_tracker_wallets_credit_transactions')
+    op.drop_index('ix_resource_tracker_wallets_credit_transactions_wallet_id', table_name='resource_tracker_wallets_credit_transactions')
+    op.drop_index('ix_resource_tracker_wallets_credit_transactions_wallet_name', table_name='resource_tracker_wallets_credit_transactions')
+    op.drop_table('resource_tracker_wallets_credit_transactions')
+
+    sa.Enum(name="transactionbillingstatus").drop(op.get_bind(), checkfirst=False)
+    sa.Enum(name="transactionclassification").drop(op.get_bind(), checkfirst=False)
+
+    # Resource tracker pricing details
+    op.add_column('resource_tracker_pricing_details', sa.Column('simcore_default', sa.Boolean(), nullable=False))
+    # op.alter_column('resource_tracker_pricing_details', 'cost_per_unit',
```

Comment:
please rm

Reply:
I will remove it in the next cleaning, now everything is green so I am merging.